pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/PatientDAOImpl.java

```diff
@@ -82,7 +80,7 @@ public PatientDTO update(PatientDTO dto) {
 		patient.setFriendlyName(dto.getFriendlyName());
 		patient.setFullName(dto.getFullName());
 		if(dto.getDateOfBirth() != null) {
-			patient.setDateOfBirth(java.sql.Date.valueOf(dto.getDateOfBirth()));
+			patient.setDateOfBirth(dto.getDateOfBirth());
```

Comment:
Why not just set the DOB to dto.getDateOfBirth()? If it returns null it's setting it to null, and if it doesn't, it just works?